src/main/java/com/auth0/jwk/GuavaCachedJwkProvider.java

```diff
@@ -34,12 +35,24 @@ public GuavaCachedJwkProvider(final JwkProvider provider) {
      * @param size        number of jwt to cache
      * @param expiresIn   amount of time a jwk will live in the cache
      * @param expiresUnit unit of the expiresIn parameter
+     * @deprecated prefer the use of {@link GuavaCachedJwkProvider#GuavaCachedJwkProvider(JwkProvider, long, Duration)}
```

Comment:
Remove this line. There's no reason to deprecate this method today. If anything, reference to it via a `@see` annotation.